client/app/components/blueprint-delete-modal/blueprint-delete-modal-service.factory.js

```diff
@@ -8,7 +8,7 @@
   function BlueprintDeleteFactory($modal) {
     var modalOpen = false;
     var modalBlueprint = {
-      showModal: showModal
+      showModal: showModal,
```

Comment:
@dtaylor113 yeah great observation! soooooo we're going to start following the greater style guide, which says multiline trailing commas they are in style! 

https://github.com/ManageIQ/manageiq/issues/8781

it looks odd at first, but the pros out weight the cons on this one... (cons being no support in ie 8/9 but who cares about that :P )